.github/workflows/codeql.yml

```diff
@@ -0,0 +1,58 @@
+name: CodeQL
+
+on:
+  pull_request:
+    branches:
+      - main
+      - release/*
+      - v0.*
+      - v1.*
+      - benchmarks
+  push:
+    branches:
+      - main
+      - release/*
+      - v0.*
+      - v1.*
```

Comment:
We don't have branches matching the `v*` pattern.

Reply:
https://github.com/open-telemetry/opentelemetry-java/branches/all?page=3

but I think you're right though that they shouldn't be needed here, let me try removing in instrumentation repo first to make sure OSSF doesn't get mad

Reply:
yup, these aren't needed

Reply:
Ah the old release strategy. Yes, we should never push to these branches.